### ISSUE TITLE
Password field shown only once and icons to reveal password "merged" into one

### ISSUE THREAD

**Issue by brozkeff** [keepassxreboot/keepassxc#5089]
## Overview
KeePassXC 2.6.0 on Linux Mint 18.3 XFce shows all icons to reveal, repeat password, generate new password etc over each other, unable to click on any of them particularly.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Left KeePassXC upgrade via apt from 2.5.x where it worked OK
2. Open any record, Same issue if language Czech, ENglish, new or compact interface

## Expected Behavior
Icons on the right of the password should be properly placed and not overlapping each other

## Actual Behavior

![2020-07-15_14-26](https://user-images.githubusercontent.com/5907371/87545106-c24ebe80-c6a7-11ea-9193-b0ef67932c17.png)

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.6.0
Revision: 0765954

Qt 5.5.1
Debugging mode is disabled.

Operating system: Linux Mint 18.3
CPU architecture: x86_64
Kernel: linux 4.15.0-106-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- Secret Service Integration

Cryptographic libraries:
 libgcrypt 1.8.1

[NOTE]: # ( Pick choices based on your environment )
Operating System: Linux
Desktop Env: XFCE
Windowing System: X11

**Comment by brozkeff** [comment]
Currently I had to manually downgrade back to 2.5.4 to have KeePassXC usable since such behaviour described above is a blocker for my use case.

**Comment by phoerious** [comment]
Please unset QT_QPA_PLATFORMTHEME. qt5ct is known to cause issues.

**Comment by droidmonkey** [comment]
You can use the keyboard shortcuts Ctrl + H (toggle pw visible) and Ctrl + G (open generator) as well. But yah, you have scaling issues that need correcting, handled in another issue.

**Comment by markbourne** [comment]
I'd also noticed this, and was about to report it before finding this existing closed issue.

In case it helps, I've found that toggling caps lock on and back off separates the icons so that they're usable. Doing that also removes the greyed-out exclamation mark at the left (with a "Warning: Caps Lock enabled" tooltip - which is confusing when caps lock isn't enabled and you don't initially realise that the icon turns red when it actually is enabled). It appears the process of updating the bar to remove that icon (when toggling caps lock back off) causes the others to be redrawn correctly.

I'm not sure if this is the same issue as resolved by unsetting `QT_QPA_PLATFORMTHEME`? It doesn't seem to be set anyway on my system (Linux Mint 18.3 with Mate desktop) and explicitly unsetting it or setting to `gtk2` doesn't make any difference to these icons. I'm not seeing the corrupted display or missing icons shown in issues 4963 and 5029 - just the ones in the password fields all on top of each other.

**Comment by droidmonkey** [comment]
Probably a Qt bug.

**Comment by markbourne** [comment]
Following discussion on #5119, at least for me this issue with the icons on top of each other is also solved with the AppImage (Qt 5.14.2) or newer Linux Mint 19 (Qt 5.9.5).

**Comment by DJCrashdummy** [comment]
well... now i upgraded to 2.6.1 (using the PPA) but this issue still exists!
i must confirm everything @markbourne wrote at https://github.com/keepassxreboot/keepassxc/issues/5089#issuecomment-660348552... although at the password generator **popup** (opened via "modify entry"), it seems that the **4 icons** on the right side of the password field are **"expanded" by default** (opposing to the password generator opened via tool bar). :+1: so it seems to be possible without the need of the clumsy "caps lock workaround". :thinking:
interestingly also **different settings** are saved **at the popup** (resp. sometimes lost there) compared to the password generator (opened via toolbar). :confused: 

**and** on top of that, what confuses me the most: usually i have **2 password files** opened by default... at the first one, in the field for the master password the 4 icons are "expanded", but at the second one they are "merged"! :shrug:  
so considering the inconsistencies it seems IMHO that this is not (only) a Qt bug!
